internal/impl/kafka/input_kafka_franz.go

```diff
@@ -67,6 +67,9 @@ Finally, it's also possible to specify an explicit offset to consume from by add
 		Field(service.NewStringField("consumer_group").
 			Description("An optional consumer group to consume as. When specified the partitions of specified topics are automatically distributed across consumers sharing a consumer group, and partition offsets are automatically committed and resumed under this name. Consumer groups are not supported when specifying explicit partitions to consume from in the `topics` field.").
 			Optional()).
+		Field(service.NewStringField("client_id").
+			Description("An identifier for the client connection.").
+			Default("timeplus")).
```

Comment:
Maybe we should keep it `benthos` here in this repo and let Neutron to always pass `timeplus` or the confluent client ID?

Reply:
Yes, if we want to contribute back, that's a good idea.

Reply:
updated it to `benthos`